cime/config/e3sm/config_grids.xml

```diff
@@ -734,6 +734,56 @@
       <mask>oQU240</mask>
     </model_grid>
 
```

Comment:
why did you add so many grids? ne8, ne25, 32,40,45...

Reply:
This was for convergence testing as a way to compare the np4 and pg2 grid cases. We don't need them, so I could take them out. I made a note about this in the description.

Reply:
not my place to say but I'd think adding grids that are not functional is not a good idea. someone may try to use them and then there will be complains?

Reply:
... also, the same is true fro domain files, NULL is better that the file that will give a mask error. You mentioned convergence, what convergence did you work on?

Reply:
ok, I'll try to remove most of them. I'll keep ne30 and ne4 both aquaplanet and F-compset. I still need to upload the files for this and add a test.

Reply:
I was testing that the solution was consistent as resolution was increased.